beps/bep_0048.rst

```diff
@@ -0,0 +1,90 @@
+:BEP: 48
+:Title: Tracker Protocol Extension: Scrape
+:Version: $Revision$
+:Last-Modified: $Date$
+:Author:  Jimmy Zelinskie <jimmyzelinskie@gmail.com>
+:Status:  Draft
+:Type:    Standards Track
+:Content-Type: text/x-rst
+:Created: 08-Sep-2016
+:Post-History:
+
+
+Abstract
+--------
+
+The original BitTorrent Protocol Specification (BEP 003) defines an exchange between a client and a tracker referred to as an Announce [#BEP_003]_.
+An Announce has a two objectives: update a peer's presence in the Swarm and discover other peers in the Swarm.
+As a result of dealing with multiple objectives, this exchange is computationally expensive when exercised at scale.
+In order to prevent trackers from becoming overloaded, announces are specified to be sent on an interval dictated by the tracker's Announce response or when a client event occurs.
+However, in order to build responsive user interfaces, clients desired a way to query metadata about Swarms in bulk in the least computationally expensive way.
+The exchange that fetches this metadata for the clients is referred to as a Scrape.
+
+
+Implementation
+--------------
+
+As the UDP Tracker Protocol Specification (BEP 015) already defines the implementation of Scrape using the UDP protocol [#BEP_015]_, the following will document the implementation for the HTTP Protocol.
+
+Changes to Announce
+...................
+
+The URL in which the Announce endpoint lives is now expected to terminate with ``/announce``.
```

Comment:
This is a bit unclear. Are you suggesting that a client should reject any announce URL that doesn't end with "/announce"? (and must do so in order to support this bep?). For deprecating existing features (which are in use), there should at least be some survey of where and how widely used it is, and what it's used for and what reasonable alternatives would be. It seems unnecessary to make such a change as part of this BEP.